### DISCUSSION TITLE
Can we add a "name" field within the "Create Cut" component?

### DISCUSSION THREAD

**Discussion by jshort45** [idea-statica/ideastatica-public#371] [Rhino/Grasshopper]
Hi,

Can we add a "name" field within the "Create Cut" component? Similar to the "create plate" component.
It would help identify the cuts a bit better when returning to IDEA.
Thanks.

![IDEA-create cut name](https://github.com/idea-statica/ideastatica-public/assets/129746855/35a0ca0e-5c47-44b2-91df-a85cfa595ccc)

**Comment by nathanlukeIDEA** [comment]
Hi @jshort45, thanks for your suggestion. We will look to improve this aspect.  In the meantime, you can use the grouping functionality in connection to group operations by member. Does this help?
![image](https://github.com/idea-statica/ideastatica-public/assets/93578548/8a1a5a6d-a302-42d8-844f-679475a6b9d7)
